spec/src/main/asciidoc/core/definition.asciidoc

```diff
@@ -493,13 +493,14 @@ Alternatively, a scope type may be specified using a stereotype annotation, as d
 
 ==== Default scope
 
-When no scope is explicitly declared by annotating the bean class or producer method or field the scope of a bean is defaulted.
+When no scope is explicitly declared by annotating the bean class or producer method or field and the bean is not deployed in a <<trimmed_bean_archive>> the scope of a bean is defaulted.
```

Comment:
I am trying to that either way, you needn't change **this spec section**. The other part of spec changes you made are sufficient.